### ISSUE TITLE
google script library

### ISSUE THREAD

**Issue by rube-de** [ccxt-dev/ccxt#230]
are there any plans that you will make ccxt available as library for google scripts?
it would be awesome if you would consider that :)

https://developers.google.com/apps-script/guides/libraries

**Comment by xpl** [comment]
I've never heard of that Google Script thing :) So it will take some time to investigate into that. Thanks for the suggestion!

**Comment by fferri** [comment]
+1

This would be nice to have, for example to fetch OHLCV data into my Google Spreadsheets where I make analysis on the altcoins markets.

**Comment by kroitor** [comment]
@fferri we're working on it (in progress), hope to deliver it soon.

**Comment by ideep** [comment]
Hi , any updates on Google App Script support ?

Thanks.

**Comment by zwarbo** [comment]
Another vote for interest in this topic. Or more information on what the gap is?

**Comment by smoochy** [comment]
Yes, please make this available. A lot of people are using Google spreadsheets to track their crypto on various exchanges.

**Comment by Aresss** [comment]
I'm also interested in this topic. Let me know if someone is already working on it and where I could support.

**Comment by npomfret** [comment]
@Aresss do you have any specific requirement?

**Comment by myn** [comment]
Bump.  Google App Script support for CCXT would be amazing..

**Comment by legitalk** [comment]
Any update on this, please?

**Comment by carlotestor** [comment]
**Triage note — can be closed without further user input**

This is a long-standing wishlist for Google Apps Script support. CCXT’s supported surfaces are the languages covered by the multi-language transpile model (currently JavaScript/TypeScript, Python, PHP, C#, Go, and Java). Native Apps Script packaging is outside that model and has not been planned.

If someone needs spreadsheet automation, a practical approach is to call a small backend (Python/JS) that uses CCXT, or use HTTP against exchange APIs directly from Apps Script.

No further clarification from the reporter is needed to close this as out of scope / not planned.

**Comment by carlotestor** [comment]
Closing without further user input: long-standing wishlist for Google Apps Script. CCXT’s supported surfaces are the languages covered by the multi-language transpile pipeline (JS/TS, Python, PHP, C#, Go, Java); a separate Apps Script port is not planned. No further reporter action needed.
